src/md/pages/documentation.hbs

```diff
@@ -2,6 +2,8 @@
 title: Documentation
 docs: [
 
+'src/computeds/md/src/Plugins/index.md',
+    'src/computeds/md/src/Plugins/AzureSAMLIntegration.md'
```

Comment:
Please add trailing comma at the end of line.

Reply:
Are you sure this section should be at the top of the documentation?

Reply:
Decided to leave integrations documentation in the current section for now.
Discussion scheduled.